CodeEdit/Features/NavigatorArea/SourceControlNavigator/Views/Repositories/SourceControlNavigatorNewBranchView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SourceControlNavigatorNewBranchView.swift
+//  CodeEdit
+//
+//  Created by Albert Vinizhanau on 10/21/23.
+//
+
+import SwiftUI
+
+struct SourceControlNavigatorNewBranchView: View {
+    @Environment(\.dismiss)
+    var dismiss
```

Comment:
```suggestion
@Environment(\.dismiss) var dismiss
```

Reply:
@activcoding 

This will show warning

```
Attributes Violation: Attributes should be on their own lines in functions and types, but on the same line as variables and imports (attributes)
```

Reply:
Oh, I forgot about SwiftLint. It can be a bit tricky to read, especially when it doesn't follow the same syntax as something like `@ObservedObject var x: X`. But I guess there's not much we can do if it's the syntax that was agreed upon.